### DISCUSSION TITLE
Tapping widget no longer opens app

### DISCUSSION THREAD

**Discussion by mkitchingh** [NightscoutFoundation/xDrip#2219] [Q&A]
I can't find any other discussion or issue on this. It started within the last few weeks, but I can't say what build for sure. I have a Pixel 6 Pro running Android 12 with the latest patch of of 7/11/22. xDrip+ is e557cba-2022.05.19. As of a few weeks ago, tapping the xDrip+ widget on my home screen no longer opens that app. It does nothing at all. I don't see any pattern. My setup is reading data from Dexcom share. I can open the app manually, and then tapping the widget again will open the app.

**Comment by Navid200** [comment]
Do you want to remove the widget from your home screen and then, add it again and see if it makes any difference?

**Reply by mkitchingh** [reply]
Sure. I just did that. I just read my initial description of the issue, and I misspoke. This doesn't happen all the time. Maybe 20% of the time. I will report back in a day or 2 with results.

**Comment by mkitchingh** [comment]
FYI, this is still happening. I had to get a new phone afterine was damaged. Another Pixel 6 pro. It didn't happen for a few weeks, but it is back. The widget is still updating, I just can't open it by tapping it. Opening xDrip manually fixes it.
